opentelemetry-extra/opentelemetry-extra.cabal

```diff
@@ -63,6 +63,7 @@ library
     http-client-tls,
     http-types,
     opentelemetry >= 0.4.0,
+    HCodecs,
```

Comment:
Please try using `binary` instead of `HCodecs`. A profiling library depending on a library for music formats seems weird.

Reply:
fix